tests/service.py

```diff
@@ -322,7 +322,7 @@ def test_delete_provider(client):
 @pytest.mark.dependency(depends=["test_delete_provider"])
 def test_empty_measurements(client):
     query = {
-        "start": "1984-04-24T00:00:00",
+        "start": "2984-04-24T00:00:00",
```

Comment:
is this change still necessary? If not, let's just revert it.

Reply:
```
test_empty_measurements(client):
    query = {
        "start": "2984-04-24T00:00:00",
    }
```
```
measurements = [
    {
        "sensor": "aqi",
        "source": "aqi",
        "description": "aqi",
        "version": None,
        "pm1dot0": 26.4,
        "pm2dot5": 26.4,
        "pm10": 26.4,
        "humidity": 100.0,
        "temperature": 134.0,
        "pressure": 1084.0,
        "co2": 1000.0,
        "longitude": -57.521369,
        "latitude": -25.194156,
        "recorded": "2020-10-24T20:47:57.370721Z",
    },
    {
        "sensor": "nullable",
        "source": "nullable",
        "description": "nullable",
        "version": "nullable",
        "pm1dot0": None,
        "pm2dot5": None,
        "pm10": None,
        "humidity": None,
        "temperature": None,
        "pressure": None,
        "co2": None,
        "longitude": -57.521369,
        "latitude": -25.194156,
        "recorded": "2020-10-24T20:47:57.370721Z",
    },
    {
        "sensor": "test",
        "source": "test",
        "description": None,
        "version": None,
        "pm1dot0": 0.0,
        "pm2dot5": 0.0,
        "pm10": 0.0,
        "humidity": 1.0,
        "temperature": -89.2,
        "pressure": 870.0,
        "co2": 200.0,
        "longitude": -57.521369,
        "latitude": -25.194156,
        "recorded": "2020-10-24T20:47:57.370721Z",
    },
]
```
If I revert, will the result be empty?

Reply:
Yes, because the previous test is deleting the provider and should delete its measurements as well, and that's we're testing :)

Reply:
I did miss this line
`@pytest.mark.dependency(depends=["test_delete_provider"])`
:)